### ISSUE TITLE
Problem with Step 6: Create a CodeDeploy application & development group.

### ISSUE THREAD

**Issue by marpierc** [airavata-courses/TeamApex#34]
I'm having trouble with Step 6 of your setup instructions, https://github.com/airavata-courses/TeamApex/wiki/Milestone-5-Guide-to-Setting-Up-Amazon's-CodeDeploy-Travis-Integration.  

The VM that I set up and instantiated in the previous steps doesn't show up in the drop down list.

**Comment by marpierc** [comment]
I found the problem. I needed to add a name to my EC2 instance.

**Comment by gouravshenoy** [comment]
Yes, CodeDeploy binds to your EC2 instance via tags. So, most preferred approach is to have a 'Name'  tag on your instance and then select that instance during CodeDeploy setup.
